### PR TITLE
[shardformer] support bias_gelu_jit_fused for models

### PR DESCRIPTION
**Pull request by flybird11111**
## 📌 Checklist before creating the PR

- [ ] I have created an issue for this PR for traceability
- [x] The title follows the standard format: `[doc/gemini/tensor/...]: A concise description`
- [ ] I have added relevant tags if possible for us to better distinguish different PRs
- [x] I have installed pre-commit: `pip install pre-commit && pre-commit install`

## 🚨 Issue number

> Link this PR to your issue with words like fixed to automatically close the linked issue upon merge
>
> e.g. `fixed #1234`, `closed #1234`, `resolved #1234`

## 📝 What does this PR do?

> Summarize your work here.
> if you have any plots/diagrams/screenshots/tables, please attach them here.

## 💥 Checklist before requesting a review

- [ ] I have linked my PR to an issue ([instruction](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue))
- [ ] My issue clearly describes the problem/feature/proposal, with diagrams/charts/table/code if possible
- [x] I have performed a self-review of my code
- [ ] I have added thorough tests.
- [ ] I have added docstrings for all the functions/methods I implemented

## ⭐️ Do you enjoy contributing to Colossal-AI?

- [ ] 🌝 Yes, I do.
- [ ] 🌚 No, I don't.

Tell us more if you don't enjoy contributing to Colossal-AI.

### CODE REVIEW

File: colossalai/shardformer/policies/bert.py [hunk 1]
```diff
@@ -38,11 +39,13 @@ def config_sanity_check(self):
 
     def preprocess(self):
         self.tie_weight = self.tie_weight_check()
+        self.enable_bias_gelu_fused = self.model.config.hidden_act == "gelu"
```
Comment (ver217) [inline colossalai/shardformer/policies/bert.py:42]: `= self.shard_config.enable_jit_fused and self.model.config.hidden_act == "gelu"`

File: colossalai/shardformer/policies/blip2.py [hunk 2]
```diff
@@ -18,12 +19,14 @@ def config_sanity_check(self):
 
     def preprocess(self):
         self.tie_weight = self.tie_weight_check()
+        self.enable_bias_gelu_fused = self.model.config.hidden_act == "gelu"
```
Comment (ver217) [inline colossalai/shardformer/policies/blip2.py:22]: same problem

File: colossalai/shardformer/policies/gpt2.py [hunk 3]
```diff
@@ -36,10 +37,11 @@ def preprocess(self):
         """
         self.tie_weight = self.tie_weight_check()
         self.origin_attn_implement = self.model.config._attn_implementation
+        self.enable_bias_gelu_fused = self.model.config.activation_function == "gelu"
```
Comment (ver217) [inline colossalai/shardformer/policies/gpt2.py:40]: same problem

File: colossalai/shardformer/policies/vit.py [hunk 4]
```diff
@@ -24,10 +25,17 @@ def config_sanity_check(self):
         pass
 
     def preprocess(self):
+        self.enable_bias_gelu_fused = self.model.config.hidden_act == "gelu"
```
Comment (ver217) [inline colossalai/shardformer/policies/vit.py:28]: same problem

File: colossalai/shardformer/policies/bert.py (hunk 1 above)
Comment (flybird11111) [reply colossalai/shardformer/policies/bert.py]: updated.

File: colossalai/shardformer/policies/blip2.py (hunk 2 above)
Comment (flybird11111) [reply colossalai/shardformer/policies/blip2.py]: updated

File: colossalai/shardformer/policies/gpt2.py (hunk 3 above)
Comment (flybird11111) [reply colossalai/shardformer/policies/gpt2.py]: updated.

File: colossalai/shardformer/policies/vit.py (hunk 4 above)
Comment (flybird11111) [reply colossalai/shardformer/policies/vit.py]: updated.

**Review by ver217 — APPROVED** [review]

**Pull request merged**